notary-server/src/error.rs

```diff
@@ -38,6 +40,11 @@ impl IntoResponse for NotaryServerError {
             bad_request_error @ NotaryServerError::BadProverRequest(_) => {
                 (StatusCode::BAD_REQUEST, bad_request_error.to_string()).into_response()
             }
+            unauthorized_request_error @ NotaryServerError::UnauthorizedProverRequest(_) => (
+                StatusCode::UNAUTHORIZED,
+                unauthorized_request_error.to_string(),
+            )
+                .into_response(),
```

Comment:
Ah that's because I wanted the error to contain the entire error string including the standard prefix [here](https://github.com/tlsnotary/tlsn/blob/dev/notary-server/src/error.rs#L18)

Reply:
ahh I see, then let's keep it.